src/converters/table-post-fixer.js

```diff
@@ -49,14 +50,14 @@ import TableWalker from './../tablewalker';
  *		<table>
  *			<thead>
  *				<tr>
- *					<td rowspan="2">FOO</td>
- *					<td colspan="2">BAR</td>
+ *					<td rowspan="2"><p>FOO</p></td>
```

Comment:
It seems that my comment disappeared so one more time...

Is it correct? Those `<p>`s won't be rendered (there will be spans or nothing). It's a bit confusing and I'm not sure if this is a mistake or a mental shortcut.

Reply:
I'd remove them.